src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -474,6 +485,18 @@ static ConfigDef newConfigDef() {
             5,
             ConfigDef.Width.NONE,
             INGESTION_METHOD_OPT)
+        .define(
+            SNOWPIPE_STREAMING_FILE_TYPE,
+            Type.STRING,
+            SNOWPIPE_STREAMING_FILE_TYPE_DEFAULT,
```

Comment:
It's hard to keep this in sync with the SDK (think about the case where we give the KC ownership to connector team in the future), I think we should just pass null here

Reply:
Null is not supported, passed `NO_DEFAULT_VALUE`